services/api-server/src/simcore_service_api_server/core/application.py

```diff
@@ -76,6 +77,9 @@ def init_app(settings: ApplicationSettings | None = None) -> FastAPI:
     if settings.SC_BOOT_MODE == BootModeEnum.DEBUG:
         remote_debug.setup(app)
 
+    if settings.API_SERVER_RABBITMQ:
```

Comment:
IMO we should not do this anymore. these variables should be required and not optional

Reply:
OK, I will make it required